package.json

```diff
@@ -106,8 +106,9 @@
 		"pegjs": "0.10.0",
 		"phpegjs": "1.0.0-beta7",
 		"postcss": "7.0.13",
-		"react-dom": "16.6.3",
-		"react-test-renderer": "16.6.3",
+		"react": "16.8.4",
+		"react-dom": "16.8.4",
+		"react-test-renderer": "16.8.4",
```

Comment:
It's surprising that `react-test-renderer` wasn't listed before 🤷‍♂️

Reply:
react wasn't listed before.

Reply:
Right :)